WooCommerce/Classes/ViewRelated/Upgrades/FullFeatureListViewModel.swift

```diff
@@ -0,0 +1,274 @@
+                                  ],
+                                  performanceFeatures: [
+                                    Localization.minMaxOrderQuantityText,
+                                    Localization.productBundlesText,
+                                    Localization.customProductKitsText,
+                                    Localization.productRecommendationsText,
+                                  ]),
+            FullFeatureListGroups(title: Localization.paymentsFeatureTitle,
+                                  essentialFeatures: [
+                                    Localization.integratedPayments,
+                                    Localization.internationalPayments,
+                                    Localization.automatedSalesTaxes,
+                                    Localization.acceptLocalPayments,
+                                    Localization.recurringPayments,
+                                  ],
+                                  performanceFeatures: []),
+
+            FullFeatureListGroups(title: Localization.marketingAndEmailFeatureTitle,
+                                  essentialFeatures: [
+                                    Localization.promoteOnTikTok,
+                                    Localization.syncWithPinterest,
+                                    Localization.connectWithFacebook,
+                                    Localization.advancedSeoTools,
+                                    Localization.advertiseOnGoogle,
+                                    Localization.customOrderEmails,
+                                  ],
+                                  performanceFeatures: [
+                                    Localization.backInStockEmailsText,
+                                    Localization.marketingAutomationText,
+                                    Localization.abandonedCartRecoveryText,
+                                    Localization.referralProgramsText,
+                                    Localization.customerBirthdayEmailsText,
+                                    Localization.loyaltyPointsProgramsText,
+                                  ]),
+
+            FullFeatureListGroups(title: Localization.shippingFeatureTitle,
+                                  essentialFeatures: [
+                                    Localization.shipmentTracking,
+                                    Localization.liveShippingRates,
+                                    Localization.printShippingLabels
+                                    ],
+                                  performanceFeatures: [
+                                    Localization.discountedShippingText,
+                                  ]),
+        ]
+    }
+}
+
+private extension FullFeatureListViewModel {
+    struct Localization {
+        static let yourStoreFeatureTitle = NSLocalizedString(
+            "Your Store",
+            comment: "The title of one of the feature groups offered with paid plans")
```

Comment:
As all these comments are basically the same, we could create a helper function to avoid duplication.